torch/jit/_fuser.py

```diff
@@ -48,6 +50,12 @@ def fuser(name):
         torch._C._jit_override_can_fuse_on_gpu(False)
         torch._C._jit_set_texpr_fuser_enabled(False)
         torch._C._jit_set_nvfuser_enabled(True)
+    elif name == 'fuser3': # oneDNN Graph
+        torch._C._jit_override_can_fuse_on_cpu(old_cpu_fuse)
```

Comment:
Similar to NNC, oneDNN graph fuser also needs to have the profiling executor turned on. You can consider adding the below lines here as well (similar to `'fuser1'`)
```python
old_profiling_executor = torch._C._jit_set_profiling_executor(True)
old_profiling_mode = torch._C._get_graph_executor_optimize(True)
```